### PR TITLE
refactor: translate password strength value & simplify AUTHENTICATION…

### PR DESCRIPTION
**Pull request by martinkaintas**

### CODE REVIEW

File: src/composables/ui.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ const hiddenCards = useStorageRef<string[]>(
 );
 const otherSettings = useStorageRef<IOtherSettings>(
   {
-    secureLoginTimeout: IS_MOBILE_APP ? AUTHENTICATION_TIMEOUTS[0] : AUTHENTICATION_TIMEOUTS[5],
+    secureLoginTimeout: IS_MOBILE_APP ? AUTHENTICATION_TIMEOUTS[0] : AUTHENTICATION_TIMEOUTS[2],
```
Comment (peronczyk) [inline src/composables/ui.ts:48]: This piece of code is repeated many times:
```
IS_MOBILE_APP ? AUTHENTICATION_TIMEOUTS[0] : AUTHENTICATION_TIMEOUTS[2]
```

I suggest to create such global constant:
```
export const  AUTHENTICATION_TIMEOUT_DEFAULT = (IS_MOBILE_APP)
  ? AUTHENTICATION_TIMEOUTS[0]
  : AUTHENTICATION_TIMEOUTS[2];
```

And reuse it.

File: src/popup/components/InputPassword.vue [hunk 2]
```diff
@@ -17,16 +17,19 @@
     <template #label-after>
       <span
         v-if="showPasswordStrength && modelValue"
-        :class="['password-strength', passwordStrength.toLowerCase()]"
-        v-text="passwordStrength"
+        class="password-strength"
+        :class="[passwordStrength]"
```
Comment (peronczyk) [inline src/popup/components/InputPassword.vue:21]: I suggest to prefix these classes:
```suggestion
        :class="[`strength-${passwordStrength}`]"
```

so the CSS block would look as follows:
```
&.strength {
  &-weak {
    ...
```

What do you think? In my opinion this gives some kind of feature grouping.

File: src/popup/pages/SecureLoginSettings.vue [hunk 3]
```diff
@@ -30,15 +30,15 @@
...
+              v-for="(ms) in AUTHENTICATION_TIMEOUTS"
+              :key="`timeout-${ms}`"
               :value="secureLoginTimeout === ms"
               :class="{ active: secureLoginTimeout === ms }"
               class="timeout"
               has-label-effect
               @input="setSecureLoginTimeout(ms)"
             >
-              <div class="row" v-text="$t('pages.secureLogin.authenticationTimeout', Number(minutes))" />
+              <div class="row" v-text="$t('pages.secureLogin.authenticationTimeout', msToMinutes(ms))" />
```
Comment (peronczyk) [inline src/popup/pages/SecureLoginSettings.vue:41]: I suggest to use `label` property instead.

Comment (martinkaintas) [reply src/popup/pages/SecureLoginSettings.vue:41]: `label` is added inside the `RadioButton.vue` component. I guess `span` tag is more appropriate.

**Review by peronczyk — APPROVED** [review]

**Pull request merged**